src/components/TagPicker/tagPickerPropTypes.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import tagPropTypes from '../tagPropTypes';
+import {iouPropTypes, iouDefaultProps} from '../../pages/iou/propTypes';
+
+const propTypes = {
+    /** The report ID of the IOU */
+    reportID: PropTypes.string.isRequired,
+
+    /** The policyID we are getting tags for */
+    policyID: PropTypes.string.isRequired,
+
+    /** The tag list we are getting tags for */
```

Comment:
it's the name right? Not the actual list itself? 
```suggestion
    /** The name of the tag list we are getting tags for */
```